apps/web/src/pages/JobPosterTemplateAdminPages/IndexJobPosterTemplatePage/IndexJobPosterTemplatePage.tsx

```diff
@@ -0,0 +1,90 @@
+import { useIntl } from "react-intl";
+import Cog8ToothIcon from "@heroicons/react/24/outline/Cog8ToothIcon";
+
+import { ROLE_NAME } from "@gc-digital-talent/auth";
+import { Container, Heading } from "@gc-digital-talent/ui";
+
+import SEO from "~/components/SEO/SEO";
+import useRoutes from "~/hooks/useRoutes";
+import useBreadcrumbs from "~/hooks/useBreadcrumbs";
+import RequireAuth from "~/components/RequireAuth/RequireAuth";
+import pageTitles from "~/messages/pageTitles";
+import Hero from "~/components/Hero";
+
+import JobPosterTemplateTableApi from "./JobPosterTemplateTable";
+
+export const IndexJobPosterTemplatePage = () => {
+  const intl = useIntl();
+  const routes = useRoutes();
+  const formattedLongPageTitle = intl.formatMessage(
+    pageTitles.indexJobPosterTemplatePageLong,
+  );
+  const formattedPageSubtitle = intl.formatMessage({
+    defaultMessage: "View, create, or edit job advertisement templates.",
+    id: "xKvQtM",
+    description: "Subtitle for the index job poster template page",
+  });
+
+  const navigationCrumbs = useBreadcrumbs({
+    crumbs: [
+      {
+        label: intl.formatMessage(pageTitles.adminDashboard),
+        url: routes.adminDashboard(),
+      },
+      {
+        label: intl.formatMessage(pageTitles.indexJobPosterTemplatePageShort),
+        url: routes.jobPosterTemplateTable(),
+      },
+    ],
+  });
+
+  return (
+    <>
+      <SEO title={formattedLongPageTitle} />
+      <Hero
+        title={formattedLongPageTitle}
+        subtitle={formattedPageSubtitle}
+        crumbs={navigationCrumbs}
+      />
+      <Container className="my-18">
```

Comment:
I'm not sure about this instance but we usually use `size="full`" to go the full width of the page for tables.

Reply:
Oh yeah, we usually get that feedback on the table pages.
[full width table](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14241/commits/5ea1d0ad0680ee13f01cf6c42c22042f5774bdbb)